src/SW_Site.c

```diff
@@ -2537,18 +2556,18 @@ void SW_SIT_init_run(
         /* Calculate bulk soil SWRCp from organic and mineral soil components */
         SWRC_bulkSoilParameters(
             SW_Site->swrc_type[s],
-            SW_Site->swrcp[s],
+            SW_Site->soils.swrcp[s],
             SW_Site->swrcpMineralSoil[s],
             SW_Site->swrcpOM,
-            SW_Site->fractionWeight_om[s],
+            SW_Site->soils.fractionWeight_om[s],
             SW_Site->depthSapric,
-            (s > 0) ? SW_Site->depths[s - 1] : 0,
-            SW_Site->depths[s]
+            (s > 0) ? SW_Site->soils.depths[s - 1] : 0,
+            SW_Site->soils.depths[s]
         );
 
         /* Check parameters of bulk soil SWRC */

```

Comment:
The `/* Check parameters of bulk soil SWRC */` should use `SW_Site->swrcp[s]` and not `SW_Site->swrcpMineralSoil[s]` -- see https://github.com/DrylandEcology/SOILWAT2/commit/f254f40d04b7200eec5229e00fcd6e34d0c5f8e1#diff-f9371e34e93e7bab2198e93bdaccdbd4186f5127f01d95aa44da7ea9d26db0e5